### DISCUSSION TITLE
settings.json

### DISCUSSION THREAD

**Discussion by Hadkins2** [codefori/vscode-ibmi#445] [Q&A] [answered]
Is there a way I can force the program to reset the settings back to default.  I have completely screwed them up and now I can't even add a connection to my iSeries.  I have tried uninstalling VS Code, but after reinstalling, all of my old extensions and the messed up settings file is back.

Thanks

Howard Adkins

**Comment by worksofliam** [comment, marked as answer]
@Hadkins2 It may also be due to an issue in the latest release. Working on perhaps undoing that now.

I don't know the method to reset it but it's absolutely possible. Porbably worth a google.

**Reply by worksofliam** [reply]
Before you do this, try 0.8.11. Then, if you still want to reset your settings, see this: https://kalkus.dev/2020/04/03/how-to-reset-settings-in-visual-studio-code/

**Reply by Hadkins2** [reply]
Got 0.8.11 and now I am able to rebuild my connections.  I will have to setup library lists, filters and profiles, but I am able to work now.  Thanks so much for your quick responses.
